README.md

```diff
@@ -8,6 +8,17 @@ VQPy is still under active development. VQPy compiler, which generates a query p
 
 ## Getting Started
 
+<details>
+<summary>Install prerequisites</summary>
```

Comment:
Shall we change it into a title for easy reference? You can add a separate section for vqpy installation